DOCUMENTATION.md

```diff
@@ -494,12 +504,28 @@ You can choose first or second class postage for your precompiled letter. Set th
 
 ### Response
 
-If the request to the client is successful, the client returns a `LetterNotificationResponse` with the `id`, `reference` and `postage` set:
+If the request to the client is successful, the client returns a `LetterNotificationResponse` with the `id`, `reference` and `postage` sets the other attributes are null:
```

Comment:
I don't understand this sentence. Does this mean that the `id`, `reference` and `postage` attributes are set to the relevant value, and the other attributes are set to null? Or that all attributes are set to null?

Reply:
Ah quite right - that doesn't sound right.

Reply:
It that better?

Reply:
👍 LGTM